logzio-sender/src/test/java/io/logz/sender/LogzioSenderTest.java

```diff
@@ -110,6 +110,31 @@ public void simpleAppending() throws Exception {
         mockListener.assertLogReceivedIs(message2, token, type, loggerName, LOGLEVEL);
     }
 
+    @Test
+    public void malformedBulk() throws Exception {
```

Comment:
This test failed before the change and now it passes?

Reply:
It's a new test